pulser/simulation/simresults.py

```diff
@@ -108,122 +350,137 @@ def get_final_state(self, reduce_to_basis: Optional[str] = None,
                                  + f"or 'digital', not '{reduce_to_basis}'.")
             ex_inds = [i for i in range(3**self._size) if ex_state in
                        np.base_repr(i, base=3).zfill(self._size)]
-            ex_probs = np.abs(final_state.extract_states(ex_inds).full()) ** 2
+            ex_probs = np.abs(final_state.extract_results(ex_inds).full()) ** 2
```

Comment:
These are `qutip` functions, you sure you want to be changing them?

Reply:
Nice catch